index.html

```diff
@@ -1827,7 +1827,7 @@ <h2 id="Collection">Backbone.Collection</h2>
 
 <pre>
 var tabs = new TabSet([tab1, tab2, tab3]);
-var spaces = new Backbone.Collection([], {
+var spaces = new Backbone.Collection(null, {
```

Comment:
Let's have the documented way be an empty array. We gracefully support falsy values.